bambam.py

```diff
@@ -48,9 +48,9 @@ def get_color():
 def load_image(fullname, colorkey = None):
     try:
         image = pygame.image.load(fullname)
-    except pygame.error, message:
-        print "Cannot load image:", fullname
-        raise SystemExit, message
+    except pygame.error as message:
+        print("Cannot load image:", fullname)
```

Comment:
AFAICT this breaks compatibility with Python 2.

Reply:
You're right, thanks